package.json

```diff
@@ -35,7 +35,7 @@
   },
   "homepage": "http://metals.rocks",
   "engines": {
-    "vscode": "^1.43.0"
+    "vscode": "^1.44.0"
```

Comment:
we should not update @types/vscode automatically in the future (that was the root cause).
I'll configure dependabot to avoid it

Reply:
Thanks!